spirit/category/admin/forms.py

```diff
@@ -33,6 +34,13 @@ def clean_parent(self):
 
             if parent and has_childrens:
                 raise forms.ValidationError(_("The category you are updating "
-                                              "can not have a parent since it has childrens"))
+                                              "can not have a parent since it has children"))
```

Comment:
Reverse this. I know it's *children*, I screwed up. But this is a `msgid`, changing it will require to update every local file. The sensitive thing to do is just translate the `en` locale (I mean add a translation for that `msgid``fix in the message).